app/views/clients/cas_readiness/_disability_verification.haml

```diff
@@ -1,15 +1,18 @@
+- description = GrdaWarehouse::Tasks::PushClientsToCas.new.description_display_for(:disability_verified_on)
+
 %section.o-section-card
   %h3 Disability Verification
   .c-card.c-card--block.c-card--padded.c-card--flush
+    - if description.present?
+      %p= description
     .d-flex.flex-wrap
       %p
         = checkmark_or_x(@client.disability_verified_on.present?)
-        Disability Verification On File
+        = GrdaWarehouse::Hud::Client.cas_columns[:disability_verified_on]
         - if @client.disability_verification_sources.exists?
           - @client.disability_verification_sources.each do |v|
             .disability_verification Disability Verified at #{v.location} on #{v.verified_at&.to_date}
         - else
-          %span= GrdaWarehouse::Hud::Client.cas_columns[:disability_verified_on]
```

Comment:
Deleted this because it was resulting in `Disability Verification On File Disability Verification On File` together with line 7

Reply:
I could be wrong, but I think that should be providing the date it was verified.

Reply:
I added the date, and also ended up changing this to be more consistent with how we display the value for consent (we say `(X) Form Not Signed` not `(X) Form Signed`).

Here are the 3 ways disability can render now (ignore the asdf nonsense, just testing a long description :)
<img width="630" alt="Screen Shot 2022-08-23 at 10 30 27 AM" src="https://user-images.githubusercontent.com/5333982/186188513-ca236505-8dcb-40db-a88e-1448adcb7dbf.png">


---

<img width="658" alt="Screen Shot 2022-08-23 at 10 30 39 AM" src="https://user-images.githubusercontent.com/5333982/186188553-a383da5f-057a-4d33-aa5b-5f1e8f24b5be.png">

---

<img width="655" alt="Screen Shot 2022-08-23 at 10 33 47 AM" src="https://user-images.githubusercontent.com/5333982/186188577-e82a3fe6-1da8-494b-9095-a6a49abdbafa.png">